src/components/element-editor/basic-details/basic-details.tsx

```diff
@@ -171,15 +171,9 @@ export const BasicDetails = () => {
                                 e: React.ChangeEvent<HTMLTextAreaElement>
                             ) => {
                                 field.onChange(e.target.value);
-                                const target = e.target;
-                                target.rows = 1;
-                                target.style.height = `${target.scrollHeight}px`;
-                                const lineHeight = 20;
-                                const newRows = Math.min(
-                                    1,
-                                    Math.floor(target.scrollHeight / lineHeight)
-                                );
-                                target.rows = newRows;
+                                const minHeight = 40;
```

Comment:
- convert `Form.Textarea` into a styled-component and apply `min-height` as a style
- move `overflow: hidden` to styled-component too